.github/workflows/test.yml

```diff
@@ -50,50 +50,4 @@ jobs:
       - run: npm install
         working-directory: netlify-functions-ecommerce
       - run: npm test
-        working-directory: netlify-functions-ecommerce
-  test-reviews:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      fail-fast: false
-      matrix:
-        node: [16]
-        os: [ubuntu-20.04]
-    name: Reviews Sample App
-    env:
-      STARGATE_BASE_URL: http://localhost:8082
-      STARGATE_AUTH_URL: http://localhost:8081/v1/auth
-      STARGATE_USERNAME: cassandra
-      STARGATE_PASSWORD: cassandra
-    services:
-      stargate:
-        image: stargateio/stargate-4_0:v1.0.65
-        env:
-          CLUSTER_NAME: stargate
-          DEVELOPER_MODE: true
-          CLUSTER_VERSION: 4.0 
-        ports:
-          - 8080:8080
-          - 8081:8081
-          - 8082:8082
-          - 9042:9042
-    steps:
-      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v3.0.2
-      - name: Setup node
-        uses: actions/setup-node@eeb10cff27034e7acf239c5d29f62154018672fd # v3.1.1
-        with:
-          node-version: ${{ matrix.node }}
-      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
-        with:
-          repository: stargate/stargate-mongoose
-          path: stargate-mongoose
-          ref: stargatification
-      - run: npm install
-        working-directory: stargate-mongoose
-      - run: npm run build
-        working-directory: stargate-mongoose
-      - run: node ../.scripts/create-tarball.js
-        working-directory: stargate-mongoose
-      - run: npm install
-        working-directory: typescript-express-reviews
-      - run: npm test
-        working-directory: typescript-express-reviews
\ No newline at end of file
+        working-directory: netlify-functions-ecommerce
```

Comment:
do we expect this to pass now?

Reply:
No. The prerequisites for the workflow are still not fixed. I just tried to remove the reference of the wip app.